docs/try-it.rst

```diff
@@ -0,0 +1,95 @@
+==========
+Test Drive
```

Comment:
Nit: The file is called `try-it.rst`, wouldn't it be consistent if this section were called "Try It"? Or vice-versa